pages/Bio-Reactor.md

```diff
@@ -0,0 +1,62 @@
+The Bio Reactor is a type of [electricity](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) generator. 
+
+## Obtaining
+A Bio Reactor can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+A Bio Reactor must be placed within 7 blocks of an [energy regulator](https://github.com/Slimefun/Slimefun4/wiki/Energy-Regulator) or a [capacitor](https://github.com/Slimefun/Slimefun4/wiki/Energy-Capacitors) in range of one (in a straight line; no diagonals), and provided with the appropriate fuel.<br>
+Once it is placed, right-click it and place fuel in its inventory (two slots on the left).
+
+## Tiers
+There is only one tier of Bio Reactor.
+
+| Tier | Energy | Buffer |
+| ---- | --- | ------ |
+| I | 8 J/s | 128 J |
+
+## Fuel
+This generator can be fueled with various vegetation.
```

Comment:
```suggestion
This generator can be fueled with various compostable items.
```
I feel like vegetation is wrong wording here, I thought flora was better but apparently theyre synonyms
(vegetation is the growth of plants over an area)
This is my suggestion towards it but if you have something

Reply:
To be fair, I had a bit of a problem with this - neither vegetation/flora works well (because Bio-Reactor uses bones, spider eyes, etc for fuel as well) nor compostable items (because again, some items won't work with either vanilla or slimefun composter).
I'm kind of inclined to just blank-card it with "various [organic?] items".